src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,67 @@
+@import "../utils/variables";
+
+@for $i from 1 through $month-length {
+  $day-number: (($i) % 7); // Use modulo to repeat every 7 days
+  $day-name: map-get($days, $day-number);
+  $numbers: map-merge($numbers, ($i: $day-name));
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 720px;
```

Comment:
You can't set width that way.
It needs to be calculated basing on number of days, calendar day size and gaps, paddings